src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header__top {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  margin: 0 50px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.logo {
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.nav__item {
+  padding: 0 20px;
+}
+
+.nav__link {
+  height: 60px;
+  text-decoration: none;
+  color: var(--nav-main-color);
```

Comment:
The `color` property uses a CSS variable `--nav-main-color`. Ensure that this variable is defined in the `:root` selector to avoid potential issues with undefined variables.